assets/css/main.scss

```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
   from { background-color: #ffb3c9}
   to {background-color: transparent}
 }
+
+/* ==== MAGICAL MODAL ==== */
+.modal {
+  align-items: center;
+  bottom: 0;
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 20;
```

Comment:
might be worth using variables for z-index and then importing , e.g. low = 0, medium = 10, high = 20 and highest = 30. Or even something like "background", "content-layer", "modal-layer" or something like that. 
That way it's easier to manage and you don't run into problems of people trying to outdo old z-indexes further down the line.